examples/Raylib/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.10)
+if (POLICY CMP0074)
+    cmake_policy(SET CMP0074 NEW)
+endif()
+
+project(LDtkLoader-Raylib)
+
+set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
```

Comment:
Setting the module path is useful only if we provide a `FindXXXX.cmake`, like in the SDL example.
Here we do not need to use it.
```suggestion
```